pwd.sh

```diff
@@ -163,9 +163,9 @@ sanity_check
 read -n 1 -p "Read, write, or delete password? (r/w/d, default: r) " action
 printf "\n"
 
-if [ "${action}" == "w" ] ; then
+if [[ "${action}" =~ ^([wW])$ ]] ; then
   create_username && write_pass
-elif [ "${action}" == "d" ] ; then
+elif [[] "${action}" =~ ^([dD])$ ]] ; then
```

Comment:
There is an extra closing bracket on this line which causes a syntax error when the script is run.